### PR TITLE
fix_color_to_#009688 and hover btn

### PR DESCRIPTION
**Pull request by hliebscher**

### CODE REVIEW

**Comment by hliebscher** [comment]
but underline fonts i don't find #00ae41

File: src/slic3r/GUI/BitmapCache.cpp [hunk 1]
```diff
@@ -322,7 +322,7 @@ wxBitmap* BitmapCache::load_svg(const std::string &bitmap_name, unsigned target_
 
     // map of color replaces
     std::map<std::string, std::string> replaces;
-    replaces["\"#00AE42\""] = "\"#009688\"";
+    replaces["\"#009688\""] = "\"#009688\"";
```
Comment (SoftFever) [inline src/slic3r/GUI/BitmapCache.cpp:325]: This shouldn't change.
I use to replace SVG files' color programmatically.

File: resources/web/guide/css/common.css [hunk 2]
```diff
@@ -124,7 +124,7 @@ body
 
 .NormalBtn:hover
 {
-	background-color:#009638;
+	background-color:#00f0d9;
```
Comment (SoftFever) [inline resources/web/guide/css/common.css:127]: Thanks for taking care the CSS ;)

**Review by SoftFever — CHANGES_REQUESTED** [review]

File: src/slic3r/GUI/BitmapCache.cpp (hunk 1 above)
Comment (hliebscher) [reply src/slic3r/GUI/BitmapCache.cpp:325]: yes  sorry !

Comment (SoftFever) [reply src/slic3r/GUI/BitmapCache.cpp]: Not at all.
I appreciate it!

**Comment by SoftFever** [comment]
Thanks!
Merged.

**Pull request merged**